components/places/sql/create_main_triggers.sql

```diff
@@ -22,3 +22,60 @@ FOR EACH ROW WHEN OLD.guid != NEW.guid
 BEGIN
     SELECT RAISE(FAIL, 'guids are immutable');
 END;
+
+-- Tags
+CREATE TEMP TRIGGER moz_tags_relations_afterinsert_trigger
+AFTER INSERT ON moz_tags_relation
+BEGIN
+  UPDATE moz_tags SET
+    lastModified = now()
+  WHERE id = NEW.tag_id;
+
+  UPDATE moz_bookmarks SET
+    syncChangeCounter = syncChangeCounter + 1
+  WHERE fk = NEW.place_id;
+
+  -- Tagging a URL increased the foreign count so that it will not be
+  -- expired or otherwise automatically removed.
+  UPDATE moz_places SET
+    foreign_count = foreign_count + 1
```

Comment:
The wipe_local, delete_everything, etc functions use this to decide whether or not to keep the place around. Do we need to tweak that?